internal/testing/e2e/e2e

```diff
@@ -142,6 +142,9 @@ duplicate_schema
 
 # Loop through ingestion commands
 for command in "${ingestion_commands[@]}"; do
+  echo @@@@ Waiting
+  sleep 60
+
   echo @@@@ Running ingestion command: "$command" | tee -a "$LOG_FILE"
   eval "$command"
   go run ${GUAC_DIR}"/cmd/guacone" certifier osv -p=false
```

Comment:
it needs to run after the `ingestion command` has completed.

Reply:
so before line 150

Reply:
Whoops... I have changed it, thanks!